Gemfile.lock

```diff
@@ -142,46 +154,6 @@ PATH
       specinfra (~> 2.10)
       syslog-logger (~> 1.6)
       uuidtools (~> 2.1.5)
-    chef (13.0.79-universal-mingw32)
```

Comment:
hrm, `rake dependencies:update_gemfile_lock` or `rake dependencies`?  anyway send me the assplosion and i could look at it...